module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -882,15 +882,15 @@ protected function getBoundWithRecords($item)
     /**
      * This method queries the ILS for holding information.
      *
-     * @param string $bibId   Bib-level id
-     * @param array  $patron  Patron login information from $this->patronLogin
-     * @param array  $options Extra options (not currently used)
+     * @param string   $bibId   Bib-level id
+     * @param array  ? $patron  Patron login information from $this->patronLogin
+     * @param array    $options Extra options (not currently used)
```

Comment:
This got formatted incorrectly for some reason; fix:

```suggestion
     * @param string $bibId   Bib-level id
     * @param ?array $patron  Patron login information from $this->patronLogin
     * @param array  $options Extra options (not currently used)
```